src/utils/sshUtils.ts

```diff
@@ -19,6 +20,7 @@ export async function getSshKey(vmName: string): Promise<string> {
     if (!await fse.pathExists(`${sshKeyPath}.pub`)) {
         // if the SSH key doesn't exist, create it
         const sshKeygenCmd: string = `ssh-keygen -t rsa -b 2048 -f ${sshKeyPath} -N ""`;
+        ext.outputChannel.appendLog(localize('generatingKey', `Generating public/private rsa key pair with command "${0}".`, sshKeygenCmd));
```

Comment:
Do we need to say "public/private rsa key pair" or could we just say "ssh key"?

Reply:
I was just mimicking what ssh-keygen says

![image](https://user-images.githubusercontent.com/5290572/75402747-85318d80-58ba-11ea-9815-a3e3552460dc.png)